packs/static-site/config.yml

```diff
@@ -0,0 +1,12 @@
+name: Static Site Pack
+
+description: Helpful configuration for anyone running a static website.
+
+level: New Relic
+
+icon: icon.jpeg
```

Comment:
Is there an icon that would fit well for this pack?
If not, can you add the newrelic icon and logo to this pack?